lib/packages/docutils/rst.nim

```diff
@@ -1937,13 +1948,33 @@ proc parseEnumList(p: var RstParser): PRstNode =
     wildToken: array[0..5, int] = [4, 3, 3, 4, 3, 3]  # number of tokens
     wildIndex: array[0..5, int] = [1, 0, 0, 1, 0, 0]
       # position of enumeration sequence (number/letter) in enumerator
-  result = newRstNodeA(p, rnEnumList)
   let col = currentTok(p).col
   var w = 0
   while w < wildcards.len:
     if match(p, p.idx, wildcards[w]): break
     inc w
   assert w < wildcards.len
+  proc checkAfterNewline(p: RstParser, report: bool): bool =
```

Comment:
regression:
```
1. hello 1
1. hello 2
1. hello 3
zook
```

before PR:
![image](https://user-images.githubusercontent.com/2194784/110692984-cdf17700-819b-11eb-9fa5-b0c13f13735c.png)


after PR:
![image](https://user-images.githubusercontent.com/2194784/110693023-d8ac0c00-819b-11eb-820b-d8488ca2610f.png)

the warning generated is good, but the output is bad